src/Theme_Command.php

```diff
@@ -899,4 +915,30 @@ private static function get_template_path( $template ) {
 
 		return $template_path;
 	}
+
+	/**
+	 * Check if current version of the theme is higher than the one available at WP.org
+	 *
+	 * @param string $slug Theme slug.
+	 * @param string $version Theme current version.
+	 *
+	 * @return bool|string
+	 */
+	protected function is_theme_version_valid( $slug, $version ) {
+		// Get Theme Info.
+		$theme_info = themes_api( 'theme_information', array( 'slug' => $slug ) );
+
+		// Return empty string for themes not on WP.org
```

Comment:
WPCS - end comments with a period:
```suggestion
		// Return empty string for themes not on WP.org.
```